flopy/utils/observationfile.py

```diff
@@ -552,6 +552,8 @@ def read_csv(fobj, dtype, delimiter=","):
         np.recarray
         """
         arr = np.genfromtxt(fobj, dtype=dtype, delimiter=delimiter)
+        if len(arr.shape) == 0:
+            arr = arr.reshape((1,))
```

Comment:
If we ever drop support for Python 3.7, I think we could use `ndmin=1` in the `np.genfromtext` call instead of manually reshaping the array. (The `ndmin` parameter was [added to `genfromtxt` in NumPy v1.23.0](https://numpy.org/doc/stable/reference/generated/numpy.genfromtxt.html), but the newest NumPy supported by Python 3.7 seems to be 1.21.6.)